### PR TITLE
Data story updates

### PR DESCRIPTION
**Pull request by aekong**
* configure data story and powerbi paragraph types for translation
* use translated titles for data story table of contents
* export other live configs

### CODE REVIEW

File: config/field.field.paragraph.document_subsection.field_title.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ bundle: document_subsection
 label: 'Title (Optional)'
 description: ''
 required: false
-translatable: true
```
Comment (zakiya) [inline config/field.field.paragraph.document_subsection.field_title.yml:20]: Was this intentional? shouldn't it be `true`

File: config/field.field.paragraph.resource_section.field_content.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ bundle: resource_section
 label: 'Resource Subsection'
 description: ''
 required: false
-translatable: true
+translatable: false
```
Comment (zakiya) [inline config/field.field.paragraph.resource_section.field_content.yml:22]: Same here.

File: config/field.field.paragraph.document_subsection.field_title.yml (hunk 1 above)
Comment (aekong) [reply config/field.field.paragraph.document_subsection.field_title.yml:20]: yes, good catch.  i made changes to those data story paragraphs on live and did an export, may have overwritten some configs that was in the `develop` branch.  thanks!

File: config/field.field.paragraph.resource_section.field_content.yml (hunk 2 above)
Comment (aekong) [reply config/field.field.paragraph.resource_section.field_content.yml:22]: resource subsection on the resources paragraph is unsupported for translation there

File: config/field.field.paragraph.resource_subsection.field_resources.yml [hunk 3]
```diff
@@ -19,7 +19,7 @@ bundle: resource_subsection
 label: Resources
 description: ''
 required: false
-translatable: true
+translatable: false
 default_value: {  }
```
Comment (zakiya) [inline config/field.field.paragraph.resource_subsection.field_resources.yml:23]: this one got switched.

**Review by zakiya — APPROVED** [review]
LGTM

**Comment by aekong** [comment]
thanks for looking!

**Pull request merged**